lib/msal-common/src/error/ServerError.ts

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { AuthError } from "./AuthError";
+
+/**
+ * Error thrown when there is an error with the server code, for example, unavailability.
+ */
+export class ServerError extends AuthError {
```

Comment:
expected that this does not have any associated errors?

Reply:
Error codes and strings for this class are set from the error sent by the server, either from the hash or in the response of the post request.